CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+This document outlines major changes between releases.
+
+## [Unreleased]
```

Comment:
Then "unreleased" can point to commits since 0.1.0, see neofs-node for example.

Reply:
I took this from neofs node, it looks exactly like they have. Not sure how should I change it.

Reply:
BTW, are you sure we need the "Unreleased" section at all? We don't have it in NeoGo and it works fine.

Reply:
I'm OK either way.

Reply:
OK, let's try to keep unreleased for now.